lib/send-mail.ts

```diff
@@ -60,15 +60,22 @@ export function parseMBox(mbox: string, gentle?: boolean):
     let subject: string | undefined;
     let to: string | undefined;
 
-    for (const line of header.split(/\n(?![ \t])/)) {
+    for (const line of header.split(/\n(?![ \t])/)
+        .map(e => e.replace("\r\n", " ").replace("\n", " "))) {
         const colon = line.indexOf(": ");
+
+        if (colon < 0 && line.lastIndexOf(":") === line.length - 1) {
+            // Keys without value are valid
+            continue;
+        }
         if (colon < 0) {
             throw new Error(`Failed to parse header line '${line}`);
         }
         const key = line.substr(0, colon);
-        const value = replaceAll(line.substr(colon + 2), "\n ", " ");
+        const value = replaceAll(line.substr(colon + 2), "\n ", " ").trim();
         switch (key.toLowerCase()) {
-            case "cc": cc = (cc || []).concat(value.split(", ")); break;
+            case "cc": cc = (cc || []).concat(value.split(", ")
+                .map(e => e.trim())); break;
```

Comment:
Could the split be done with a regex to include whitespace+ and avoid the map?  There is no trailing whitespace at this point.

Just curious, what mail server/agent is including trailing whitespace?  Have not reviewed the RFCs on what is expected;

Reply:
Yes, there's no trailing whitespace, but I have encountered cases with leading whitespace after the split.

Reply:
Split with regex /,\s+/?

Reply:
I'm totally fine when you just close the PR and do the changes to your liking ;-)

It's not worth the time for those little changes; anyway you won't see me using regex as long as it's avoidable, but just go ahead, I don't mind how you do it, neither when it's authored by you.

Reply:
> I'm totally fine when you just close the PR and do the changes to your liking ;-)

I don't want to close it.  I would like to have test cases with the problems you have encountered so they do not get regressed.  You have the knowledge in this area.

Reply:
OK, will do.

Reply:
I am looking forward to the updates!

Reply:
> Split with regex /,\s+/?

For extra goodness, we could use `/,\s*/`, I think. The only remaining problem would be that something like this would not be handled correctly:

```
Cc: "Schindelin, Johannes" <johannes.schindelin@gmx.de>
```

But that's not a new problem, and therefore not the responsibility of this PR to fix.

Reply:
I don't like regex...

Reply:
In this instance, it seems to be the best tool for the job, though.